src/analysis/analysis.jl

```diff
@@ -93,15 +101,16 @@ Computes confidence interval for series of data.
 # Returns
 Confidence interval (lower bound, median and higher bound) for each series step
 """
-function series_confint(data::AbstractMatrix; agg_dim::Symbol=:scenarios)::Matrix{Float64}
-    slice_dim = data isa NamedDimsArray ? NamedDims.dim(data, agg_dim) : 2
-    return quantile.(Slices(data, slice_dim), [0.025 0.5 0.975])
+function series_confint(data::AbstractMatrix; agg_dim::Symbol = :scenarios)::Matrix{Float64}
+	slice_dim = data isa NamedDimsArray ? NamedDims.dim(data, agg_dim) : 2
+	return quantile.(Slices(data, slice_dim), [0.025 0.5 0.975])
 end
 
 include("pareto.jl")
 include("intervention.jl")
 include("clustering.jl")
 include("rule_extraction.jl")
 include("scenario.jl")
+include("decision.jl")
```

Comment:
Mostly because the function uses the result set so I thought it was better suited to analysis, but you're right it's not ideal. What do you think @ConnectedSystems ?